Include/internal/pycore_unicodeobject.h

```diff
@@ -77,8 +77,6 @@ _PyUnicodeWriter_WriteCharInline(_PyUnicodeWriter *writer, Py_UCS4 ch)
 
 /* --- Characters Type APIs ----------------------------------------------- */
 
-extern int _PyUnicode_IsXidStart(Py_UCS4 ch);
-extern int _PyUnicode_IsXidContinue(Py_UCS4 ch);
 extern int _PyUnicode_ToLowerFull(Py_UCS4 ch, Py_UCS4 *res);
 extern int _PyUnicode_ToTitleFull(Py_UCS4 ch, Py_UCS4 *res);
 extern int _PyUnicode_ToUpperFull(Py_UCS4 ch, Py_UCS4 *res);
```

Comment:
Forgot to mention: These should also be moved to the new pycore_unicodectype.h file (it should collect all internal APIs defined in unicodectype.c).